packages/tests/README.md

```diff
@@ -0,0 +1,48 @@
+# Description
+
+This package contains tests for the `js-waku` library.
+Some of the tests are integration tests that require a running an `nwaku` or a `go-waku` node. These nodes are setup to be run in a docker container.
+
+# Pre-requisites
+
+Some of the tests from this package require a running `nwaku` or `go-waku` node. These nodes are setup to be run in a docker container.
+Therefore, you need to have `docker` installed on your machine to run the tests.
+
+# Running interop tests
+
+- The tests by default run against an `nwaku` node with the image name specified in `nwaku.ts` and `packages/tests/package.json`. The tests can be run against a different image by setting the environment variable `WAKUNODE_IMAGE` to the desired image.
+
+- The tests can be run against a `go-waku` node by setting the environment variable `WAKUNODE_IMAGE` to the desired `go-waku` image.
+
+:::info
```

Comment:
```suggestion
```

This is `mdx` syntax. It does not work here. Please check your md files rendering in github.